pos_ticket_without_price/static/src/js/screens.js

```diff
@@ -28,8 +28,14 @@ odoo.define('pos_ticket_without_price.screens', function (require) {
             var button_print_ticket_without_price =
                 this.$('.button.print_ticket_without_price');
             button_print_ticket_without_price.click(function () {
-                self.render_ticket_without_price();
-                self.print();
+                console.log("self.pos.config.iface_print_via_proxy")
+                console.log(self.pos.config.iface_print_via_proxy)
```

Comment:
as @legalsylvain requested, these lines should be removed

Reply:
yes sorry its my first contribution and i have thinks to improve . Really thx.